src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -210,7 +211,14 @@
                     throw Error(options.unknownErrorMessage);
                 })
                 .then(function (data) {
-                    if (!!data.errorMessage) {
+                    if (data.hintCode === 'StartFailed') {
```

Comment:
I would move this logic to the server. Then it's easier to extend.
Return a bool, something like `retry`. And check the hint code in the backend instead.

Reply:
Wouldn't that mean moving state to the server? So when there are multiple nodes behind a load balancer they will not know how many times they retried already?

Reply:
Good idea!